// sports-odds-api/resources/index.js
"use strict";
// File generated from our OpenAPI spec by Stainless. See CONTRIBUTING.md for details.
Object.defineProperty(exports, "__esModule", { value: true });
exports.Teams = exports.Stream = exports.Stats = exports.Sports = exports.Players = exports.Leagues = exports.Events = exports.Account = void 0;
var account_1 = require("./account.js");
Object.defineProperty(exports, "Account", { enumerable: true, get: function () { return account_1.Account; } });
var events_1 = require("./events.js");
Object.defineProperty(exports, "Events", { enumerable: true, get: function () { return events_1.Events; } });
var leagues_1 = require("./leagues.js");
Object.defineProperty(exports, "Leagues", { enumerable: true, get: function () { return leagues_1.Leagues; } });
var players_1 = require("./players.js");
Object.defineProperty(exports, "Players", { enumerable: true, get: function () { return players_1.Players; } });
var sports_1 = require("./sports.js");
Object.defineProperty(exports, "Sports", { enumerable: true, get: function () { return sports_1.Sports; } });
var stats_1 = require("./stats.js");
Object.defineProperty(exports, "Stats", { enumerable: true, get: function () { return stats_1.Stats; } });
var stream_1 = require("./stream.js");
Object.defineProperty(exports, "Stream", { enumerable: true, get: function () { return stream_1.Stream; } });
var teams_1 = require("./teams.js");
Object.defineProperty(exports, "Teams", { enumerable: true, get: function () { return teams_1.Teams; } });
